docs/pages/database-access/reference/aws.mdx

```diff
@@ -52,3 +55,50 @@ arn:aws:rds-db:<region>:<account-id>:dbuser:<resource-id>/<db-user>
 
 See [Creating and using an IAM policy for IAM database access](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/UsingWithRDS.IAMDBAuth.IAMPolicy.html)
 for more information.
+
+## Redshift policy
+Teleport uses temporary credentials generated by AWS to authenticate with
+Redshift databases.
+
+In order to authorize Teleport to generate temporary IAM tokens, create an IAM
+role with the `GetClusterCredentials` permission:
+
+<Details title="Example IAM policy document" opened={false}>
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "redshift:GetClusterCredentials",
+            "Resource": [
+                "arn:aws:redshift:*:1234567890:dbuser:*/*",
+                "arn:aws:redshift:*:1234567890:dbname:*/*",
+                "arn:aws:redshift:*:1234567890:dbgroup:*/*"
+            ]
+        }
+    ]
+}
+```
+</Details>
+
+The resource ARN string has the following format:
+
+```
+arn:aws:redshift:<region>:<account-id>:<resource>:<cluster-id>/<name>
+```
+
+Parameters:
+
+- `region`: AWS region where your Redshift cluster is deployed, or a wildcard.
+- `account-id`: ID of the AWS account where the Redshift cluster is deployed.
+- `resource`: One of `dbuser`, `dbname` or `dbgroup` to restrict access to
+  database accounts, names or groups respectively.
+- `cluster-id`: Redshift cluster identifier, or a wildcard.
+- `name`: Name of a particular database account, name or group (depending
+  on the `resource`), or a wildcard.
```

Comment:
Can you format this as a markdown table so it's consistent with RDS/Aurora section above?